tests/default/flow_framework/workflow.yaml

```diff
@@ -0,0 +1,59 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+description: Test flow_framework endpoints.
+version: '>= 2.12'
+epilogues:
+  - path: /_plugins/_flow_framework/workflow/{workflow_id}
+    method: DELETE
+    status: [200, 404]
+    parameters:
+      workflow_id: ${create_flow_framework.test_workflow_id}
+chapters:
+  - synopsis: Create workflow.
+    id: create_flow_framework
+    path: /_plugins/_flow_framework/workflow
+    method: POST
+    request:
+      payload:
+        name: test_create_work_flow
+    response:
+      status: 201
+    output:
+      test_workflow_id: payload.workflow_id
```

Comment:
Could add some invalid JSON here that would fail parsing for another test.

Reply:
Will add in the next pr